cmd/main.go

```diff
@@ -50,8 +52,10 @@ var (
 	bqDataset   = flag.String("bq_dataset", "", "DataCommons BigQuery dataset.")
 	schemaPath  = flag.String("schema_path", "", "The directory that contains the schema mapping files")
 	// Base Bigtable Cache
-	useBaseBt     = flag.Bool("use_base_bt", true, "Use base bigtable cache")
-	baseTableName = flag.String("base_table", "", "Base cache Bigtable table.")
+	useBaseBt         = flag.Bool("use_base_bt", true, "Use base bigtable cache")
+	baseTableName     = flag.String("base_table", "", "Base cache Bigtable table.")
+	useImportGroup    = flag.Bool("use_import_group", false, "Use multiple base tables from import group")
```

Comment:
Document in description that this mode is relevant only when --use_base_bt is true? (Unless I read the implementation, that was not clear)

Reply:
Done